superset-frontend/src/components/TimezoneSelector/TimezoneSelector.test.tsx

```diff
@@ -27,91 +27,104 @@ jest.spyOn(moment.tz, 'guess').mockReturnValue('America/New_York');
 const getSelectOptions = () =>
   waitFor(() => document.querySelectorAll('.ant-select-item-option-content'));
 
-it('use the timezone from `moment` if no timezone provided', () => {
-  const onTimezoneChange = jest.fn();
-  render(<TimezoneSelector onTimezoneChange={onTimezoneChange} />);
-  expect(onTimezoneChange).toHaveBeenCalledTimes(1);
-  expect(onTimezoneChange).toHaveBeenCalledWith('America/Nassau');
-});
+describe('TimezoneSelector', () => {
+  beforeAll(() => {
+    jest.useFakeTimers('modern');
+    jest.setSystemTime(new Date('January 10 2022'));
```

Comment:
This sets/freezes the system time so that tests will be run under the same conditions every time.